bounded-collections/src/lib.rs

```diff
@@ -76,6 +97,16 @@ macro_rules! impl_const_get {
 				Some(T)
 			}
 		}
+		impl<const T: $t> ConstGet<$t> for $name<T> {
+			fn get() -> $t {
+				T
+			}
+		}
+		impl<const T: $t> ConstGet<Option<$t>> for $name<T> {
+			fn get() -> Option<$t> {
+				Some(T)
```

Comment:
Why do we need this implementation?

Reply:
It actually implements `ConstGet` for all `const` types.

Reply:
I meant why do we need an implementation of `ConstGet<Option<T>>` on the `Const*` types.

Reply:
Ah, I was following a similar paradigm as `Get<Option<T>>` that has already been implemented for const types.